### PR TITLE
10.0 mig account move line payable receivable filter

### PR DESCRIPTION
**Pull request by mourad-ehm**

### CODE REVIEW

File: account_move_line_payable_receivable_filter/README.rst [hunk 1]
```diff
@@ -1,56 +1,56 @@
...
-For further information, please visit:
-
- * https://www.odoo.com/forum/help-1
+#. Go to ...
 
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/account-financial-tools/10.0
 
+.. repo_id is available in https://github.com/OCA/maintainer-tools/blob/master/tools/repos_with_ids.txt
```
Comment (pedrobaeza) [inline account_move_line_payable_receivable_filter/README.rst]: You can remove this part.

File: account_move_line_payable_receivable_filter/views/account_move_line_view.xml [hunk 2]
```diff
@@ -2,13 +2,12 @@
 <openerp>
```
Comment (pedrobaeza) [inline account_move_line_payable_receivable_filter/views/account_move_line_view.xml]: Put `<odoo>` and remove `<data>` tag

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
An please squash together both "OCA Transbot" commits following https://github.com/OCA/maintainer-tools/wiki/Merge-commits-in-pull-requests

File: account_move_line_payable_receivable_filter/views/account_move_line_view.xml (hunk 2 above)
Comment (mourad-ehm) [reply account_move_line_payable_receivable_filter/views/account_move_line_view.xml]: OK. it is an oversight.

**Review by pedrobaeza — APPROVED** [review]

**Comment by mourad-ehm** [comment]
thanks @pedrobaeza  for review.

**Review by sbidoul — APPROVED** [review]
code review, no test

**Review by legalsylvain — APPROVED** [review]
Thanks.

**Pull request merged**